renderer/pages/loader/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+import { QuestionList } from '../../models/QuestionList';
+import { Question } from '../../models/Question';
+
+export default function Loader() {
+  const [loadedQuestions, setLoadedQuestions] = useState<Question[]>([]);
+
+  const questionList = QuestionList.getInstance();
+
+  useEffect(() => {
+    loadQuestionsFromList();
+  }, []);
+
+  const loadQuestionsFromList = () => {
+    const allQuestions = questionList.getAllQuestions();
+    setLoadedQuestions(allQuestions);
+  };
+
+  const downloadQuestionsToFile = () => {
+    try {
+      const questions = questionList.unusedQuestions.concat(questionList.usedQuestions);
+      const jsonData = JSON.stringify(questions, null, 2);
+      const blob = new Blob([jsonData], { type: 'application/json' });
+      const url = URL.createObjectURL(blob);
+      const a = document.createElement('a');
+      a.href = url;
+      a.download = 'questions.json';
+      document.body.appendChild(a);
+      a.click();
+      document.body.removeChild(a);
+      URL.revokeObjectURL(url);
+    } catch (error) {
+      console.error('Error saving questions to file:', error);
+    }
+  };
+
+  const uploadQuestionsFromFile = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+    if (!file) return;
+    const reader = new FileReader();
+    reader.onload = (event) => {
+      try {
+        questionList.usedQuestions = [];
+        questionList.unusedQuestions = [];
+        const content = event.target?.result?.toString(); // Convert ArrayBuffer to string
+        if (content) {
+          const questions = JSON.parse(content);
+          const questionObjects = questions.map((question: any) => {
```

Comment:
pls no any types, also change tsconfig to strict: true, I will later update the development branch